catala.opam

```diff
@@ -37,7 +37,9 @@ depends: [
   "js_of_ocaml-ppx" {>= "3.8.0"}
   "camomile" {>= "1.0.2"}
   "cppo" {>= "1"}
+  "obelisk" {dev}
```

Comment:
Be careful when added these lines, the `opam` team asked me to remove them last time I published: https://github.com/ocaml/opam-repository/pull/20861#issuecomment-1061807613